docs/application/dotnet/get-started/nui/quickstart.md

```diff
@@ -0,0 +1,291 @@
+
+        ```
+        app.Run(args);
+        ```
+
+### Build and Run NUI Application
+1.  Building the solution:
+
+    - In the Visual Studio menu, select **Build** > **Build Solution**.
+
+    - In the Solution Explorer view, right-click the solution name and select **Build**.
+
+2.  Launching Tizen Emulator:
+
+    - In the Visual Studio menu, select **Tools &gt; Tizen &gt; Tizen Emulator Manager**.
+
+    - Alternatively, click **Launch Tizen Emulator** in the Visual Studio toolbar to launch the Tizen Emulator Manager:
+
+      ![launch_emulator1](media/tizen_project_wizard_capture_template3.png)
+
+    - In the Emulator Manager, select an emulator from the list, and click **Launch**:
+
+      ![launch_emulator2](media/tizen_project_wizard_capture_template4.png)
+
+    - You must update the following emulator images to use the latest version.
+
+3.  After you launch an emulator instance, you can select the target from the drop-down list to change the deployment target in the Visual Studio toolbar:
+
+    ![VS_Toolbar2](media/tizen_project_wizard_capture_template6.png)
+
+    You can deploy your NUI app to the target emulator in one of the following ways:
+
+    1. Press the **F5** key or **Ctrl+F5** key.
+
+    2. In the Visual Studio menu, select **Debug > Start Debugging** or **Start Without Debugging**.
+
+    3. On the Visual Studio toolbar, click an emulator instance:
+
+       ![VS_Toolbar](media/tizen_project_wizard_capture_template5.png)
+
+4.  If the deployment is successful, then the application is launched on the target emulator.
+
+    The following are snapshots of launching this NUI app on mobile emulators:
+
+    **Figure: NUI app on Mobile emulator**
+
+    ![NUIAppOnMobile](./media/nui-hello-world.png)
+
+    Visual Studio uses the Smart Development Bridge (SDB) to communicate with the target device or emulator.
+
+    You can also deploy your NUI application with SDB manually:
+
+    1.  Copy the application .tpk file from the project binary path to the SDB tool path:
+
+        - The `SampleNUIApp` tpk file(`org.tizen.example.SampleNUIApp-1.0.0.tpk`) is located in `ProjectPath\SampleNUIApp\SampleNUIApp\bin\Debug\tizen50`.
+
+        - To locate the project path in the Solution Explorer view, right-click the solution name and click **Open Folder in File Explorer**.
+
+        - SDB tool is located in `c:\tizen\tools\sdb.exe`.
+
+    2.  Launch the Tizen SDB command prompt:
+
+        - For Windows OS, launch the Tizen SDB command prompt (**Tool** > **Tizen** > **Tizen Sdb Command Prompt**).
+
+        - For Linux OS, you can use the SDB command directly in your project path.
+
+    3.  Install your application using the following SDB command:
+
+        ```
+        sdb install org.tizen.example.SampleNUIApp-1.0.0.tpk
+        ```
+
+
+## Related Information
```

Comment:
For the Get Started pages, we don't attach this "Related Information" section. 
How about adding a note instead of "Related Information" at the starting point of this page to indicate that NUI Applications are supported since Tizen 4.0?

Reply:
Modified.